discovery-rules/attachment_any_eml.yml

```diff
@@ -0,0 +1,12 @@
+name: "Attachment: Any eml file"
+description: |
+  Potential EML file in message. This rule may potentially be combined with a webhook 
+  notification to submit the EML contents to the analysis API
```

Comment:
```suggestion
  Any EML attachment. This rule can be combined with a webhook action for further analysis
  of attached EML files, eg via the analysis API.
```

Reply:
small nit: notification -> action, and slight verbage change since you'll technically have to poll instead of sending the contents directly

Reply:
Changed